includes/wikia/traits/TitleTrait.php

```diff
@@ -34,7 +34,7 @@ public function getIndirectLinks( $level = 0 ) {
 
 		// Make the query
 		$plConds = [
-			'page_id=pl_from',
+			'page_id' => 'pl_from',
```

Comment:
This won't work as it will become `page_id = 'pl_from'` rather than `page_id = pl_from` since the values in key value pairs get run through `addQuotes`.